ai-ml/trainium-inferentia/examples/llama2/install-pre-requsites-for-ec2.sh

```diff
@@ -4,12 +4,12 @@
 install_docker() {
     echo "Checking and installing Docker..."
     sudo yum install docker -y
-    sudo systemctl start docker
```

Comment:
Why'd you change this? The existing code should be correct.

Reply:
@sanjeevrg89 Could you please verify the line that you removed and update accordingly